cli/cmd/plugin/unmanaged-cluster/config/config.go

```diff
@@ -60,8 +61,10 @@ type PortMap struct {
 type UnmanagedClusterConfig struct {
 	// ClusterName is the name of the cluster.
 	ClusterName string `yaml:"ClusterName"`
-	// KubeconfigPath is the serialized path to the kubeconfig to use.
+	// KubeconfigPath is the serialized path to the kubeconfig.
 	KubeconfigPath string `yaml:"KubeconfigPath"`
```

Comment:
Initially, I had expected this to be removed in favor of `ExistingClusterKubeconfig`.

Any value in keeping it?

Reply:
> ExistingClusterKubeconfig is almost a sort of "subset" of KubeconfigPath

Hmmm maybe it can be done with just one then. Might be worth chatting over further